woocommerce/woocommerce-android
# Interac refund Client Side Logic

part of: #5711 



### Description

This PR handles core [client-side Interac refund logic](https://stripe.com/docs/terminal/features/refunds#client-side). This PR doesn't contain API-specific changes for Interac refund and just contains the logic necessary to process the Interac refund on the client-side successfully.


### Testing instructions

Green CI should be enough. In order to see what the end product looks like, please refer to PR https://github.com/woocommerce/woocommerce-android/pull/6157.
